.circleci/config.yml

```
@@ -87,6 +122,9 @@ jobs:
       - git_checkout_from_cache
       - npm_install
       - build
+      - compress
+      - upload_artifact
```

```suggestion
       # Uploading the built code to s3 to create a backup of key services separate from Kubernetes deployment.
      - upload_artifact
```
Adding a comment to let the future developers know of the reason for this.